frontend/src/static/js/v_summary.js

```diff
@@ -39,6 +24,26 @@ function dateRounding(datestr, roundDown) {
   return window.getDateStr(datems);
 }
 
+window.deactivateAllOverlays = function deactivateAllOverlays() {
```

Comment:
The moving of these two methods are not needed for this PR right?

Reply:
just want to group all window method together and function together.
I need to have getHex before window.getFontColor, but then I will have sth like
```
function getHex
window.func
window.func
function getDate
```

Reply:
I think this can be in another PR as these two methods are unrelated to this PR. We usually do not modify code that is not related to the PR.